grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -441,7 +449,7 @@ private void invokeOnMessage(I request) {
             listener.onMessage(request);
         } catch (Throwable t) {
             upstream.cancel();
-            close(GrpcStatus.fromThrowable(t), new Metadata());
+            close(convertThrowableToStatus(t), new Metadata());
```

Comment:
How about adding `close(Throwable, Metadata)` to deduplicate the `convertThrowableToStatus()` calls?